src/tests/system/tests/test_ipa.py

```diff
@@ -127,6 +127,87 @@ def test_ipa__hostpublickeys_by_ip(client: Client, ipa: IPA, public_keys: list[s
         assert f"{ip} {key}" in result.stdout_lines, "Did not get expected public keys!"
 
 
+@pytest.mark.ticket(gh=7583)
+@pytest.mark.importance("low")
+@pytest.mark.parametrize("option, output", [(None, "[{hostname}]:{port} {key}"), ("-o", "[{hostname}] {key}")])
```

Comment:
The hostname alone will not have brackets.

Reply:
updated.